### PR TITLE
added 1.15.2 PC files

### PR DESCRIPTION
**Pull request by uncovery**

### CODE REVIEW

**Comment by rom1504** [comment]
Can you add them to loadpaths.js too ?

**Comment by uncovery** [comment]
Is that what you meant?

File: data/dataPaths.json [hunk 1]
```diff
@@ -465,7 +465,11 @@
     "1.15.2": {
       "protocol": "pc/1.15.2",
       "version": "pc/1.15.2",
-      "language": "pc/1.15.2"
+      "language": "pc/1.15.2",
+      "blocks": "pc/1.15.2",
+      "entities": "pc/1.15.2",
+      "items": "pc/1.15.2",
+      "recipies": "pc/1.15.2"
```
Comment (rom1504) [inline data/dataPaths.json:472]: Typo

Comment (rom1504) [reply data/dataPaths.json:472]: Biomes missing

Comment (rom1504) [reply data/dataPaths.json:472]: You didn't add blocks

**Comment by uncovery** [comment]
hmmm it seems I do not know how this works. I detected those changes already and thought I had added them. Will try to do again... sorry about this.

**Comment by rom1504** [comment]
you don't have to close it, you can add more commits

**Comment by uncovery** [comment]
Yeah that's what I tried last time, and since that somehow failed, I thought I'd rather start from scratch... just did a new one, that looks fine.

**Pull request closed without merging**